core/domain/entities/routing/handlers/shared/RegularRequests.php

```diff
@@ -27,8 +27,9 @@ public function initialize()
     {
         $basic_nodes = [
             'EventEspresso\core\domain\entities\routing\data_nodes\core\Api',
-            'EventEspresso\core\domain\entities\routing\data_nodes\core\CurrentUser',
+            'EventEspresso\core\domain\entities\routing\data_nodes\core\Capabilities',
             'EventEspresso\core\domain\entities\routing\data_nodes\core\Locale',
+            'EventEspresso\core\domain\entities\routing\data_nodes\core\SitePermissions',
             'EventEspresso\core\domain\entities\routing\data_nodes\core\SiteUrls',
```

Comment:
Why `CurrentUser` is removed?

Reply:
this was just an array of nodes that all have the same default dependencies.
The CurrentUser class had the Capabilities class added as a dependency so that had to be set up separately now that it no longer has the default dependencies.